PluginCore/PluginCore/FRService/FRSearch.cs

```diff
@@ -608,6 +612,61 @@ private List<SearchMatch> SearchSource(string src, int startIndex, int startLine
             return results;
         }
 
+        private bool LookupRegex(string ba, ref int i)
+        {
+            if (SourceFile == null)
+                return false;
+
+            string ext = System.IO.Path.GetExtension(SourceFile).ToLowerInvariant();
+
+            // Haxe, ActionScript, and JavaScript support Regex literals
+            if (!Helpers.FileInspector.IsActionScript(SourceFile, ext) &&
```

Comment:
Actually, although passing `SourceFile` is one valid approach, I think it's not the right one as it doesn't support LoomScript files. Which means we have to hardcode more and more file extensions here.

Not sure what should be the right approach - we could check the Scintilla syntax configuration (does it have regexes?), but ultimately I think we should just look up any `/.../` as regexes no matter what the language and including Haxe where a valid regex has to start with `~`.